front-end/src/components/UserForm/style.scss

```diff
@@ -108,7 +104,14 @@
   cursor: pointer;
 }
 
-.show-hide-btn{
-  font-weight: bold;
-  cursor: pointer;
+.sign-up-bottom-paragraph-container {
```

Comment:
You're right. I decided to give them that prefix because maybe it would be easier for you to identify which styles belong to which form. But I'll fix it. Thanks bro.